packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -132,8 +134,8 @@ const AccountSwitcher = props => {
         }
     };
 
-    const redirectToMt5Real = () => {
-        redirectToMt5('real');
+    const redirectToMt5Real = market_type => {
+        redirectToMt5(`real${market_type}`);
```

Comment:
```suggestion
        redirectToMt5(`real-${market_type}`);
```
can we have `-` in here, and just pass the `market_type` without `-` to `redirectToMt5Real`, `redirectToMt5Demo`, and `redirectToDXTradeDemo` functions